setup.py

```diff
@@ -64,16 +64,21 @@
     extras_require['all'].extend(reqs)
 
 install_requires = [
-    'Invenio[base,auth,files]{}'.format(invenio_version),
+    'Invenio[base,auth]{}'.format(invenio_version),
+    'invenio-rdm-records>=0.9.0',
+    'invenio-records-permissions>=0.7.0',
```

Comment:
(I know these dependencies have only been moved.) Why was it that we didn't cap these version specifiers? (Given the approach we have taken to dependencies so far, we may want to...)

Reply:
True that. Fixed :)